codedigger/atcoder/serializers.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework import serializers
+from problem.models import Problem, atcoder_contest
+
+
+class AtcoderProblemSerializer(serializers.ModelSerializer):
+
+    status = serializers.SerializerMethodField()
+
+    def get_status(self, obj):
+
+        if obj.prob_id in self.context.get('solved'):
+            return 'solved'
+        elif obj.prob_id in self.context.get('wrong'):
+            return 'wrong'
+        else:
+            return 'not_attempt'
+
+    class Meta:
+        model = Problem
+        fields = [
+            'name', 'url', 'prob_id', 'tags', 'contest_id', 'rating', 'index',
+            'platform', 'difficulty', 'editorial', 'status'
+        ]
+
+
+class AtcoderUpsolveContestSerializer(serializers.ModelSerializer):
+
+    problems = serializers.SerializerMethodField()
```

Comment:
Same for this function. Duplicate hai toh purane ko remove kardo, jha jha purana use kiya hua hai vaha vaha isko import karao.